src/main/java/chessmaster/user/Player.java

```diff
@@ -72,6 +72,27 @@ public void printAllPieces() {
         }
     }
 
+    /**
+     * Converts the information in the moves array into a String, to be used in saving the game.
+     *
+     * @return String containing information about the player's past moves
+     */
+    public String movesToString() {
+        String out = new String();
+
+        for (int i = 0; i < moves.size(); i = i + 1) {
+            Move move = moves.get(i);
+
+            if (i < moves.size() - 1) {
+                out = out + move.toFileString() + ", ";
```

Comment:
Would be good to declare the public static final moves delimiter variable once to prevent any future errors 
e.g. `MOVES_DELIMITER`

Reply:
Not a final variable since the number of moves will change as the game goes on